src/ansys/fluent/core/utils/deprecate_args.py

```diff
@@ -0,0 +1,33 @@
+"""Module that provides a method to handle deprecated arguments."""
+
+import functools
+import warnings
+
+
+def deprecate_argument(
+    old_arg, new_arg, converter, deprecation_class=DeprecationWarning
+):
+    """Warns user that the argument provided is deprecated, and automatically replaces
+    the deprecated argument with the appropriate new argument."""
+
+    def decorator(func):
+        """Holds the original method to perform operations on it."""
+
+        @functools.wraps(func)
+        def wrapper(*args, **kwargs):
+            """Warns about the deprecated argument and replaces it with the new
+            argument."""
+            if old_arg in kwargs:
+                warnings.warn(
+                    f"'{old_arg}' is deprecated. Use '{new_arg}' instead.",
```

Comment:
At this point we will be automatically replacing the deprecated argument with the new one, right? So I think a more appropriate wording would be:
```suggestion
                    f"'{old_arg}' is deprecated. Using '{new_arg}' instead.",
```
And can we move `val` definition before this warning, so that we can show it in the warning as well to better inform the user?

For example, putting these two suggestions together:
```suggestion
                    f"'{old_arg}' is deprecated. Using '{new_arg} = {val}' instead.",
```